src/schemas/receiver.test.js

```diff
@@ -111,7 +121,7 @@ describe('Receiver Validation', () => {
     const { error } = validate(receiver, receipt)
     expect(error).toBeDefined()
     expect(error.message).toBe(
-      '"receiver.authorisationNumbers" must contain at least 1 items'
+      'At least one site authorisation number is required'
```

Comment:
let's keep the old error message, it's formatted like the original error messages and it's clearer for developers to address